object/object.go

```diff
@@ -16,6 +16,15 @@ type EncryptionKey struct {
 	entropy *[32]byte
 }
 
+// String implements fmt.Stringer.
+func (k EncryptionKey) String() string {
```

Comment:
(nit) you could move the implementation of `MarshalText` here and then call `String` in `MarshalText` to avoid the panic.

Reply:
haha yeah that's way better